app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def self.update_comments_counter(post)
+    counter = Comment.where("post_id = #{post.id}").count
+    post.comments_counter = counter
+    post.save
+  end
+end
```

Comment:
- [ ] Comment model should have a method that updates the comments counter for a post so would you add that method as well, because it makes the app efficient and user friendly!

Reply:
:+1: This method already exists @sevinchek , that was a mistake, your project was worthy approval. Sorry for this @sevinchek